mm2src/mm2_main/src/lp_ordermatch.rs

```diff
@@ -1399,6 +1410,13 @@ impl<'a> TakerOrderBuilder<'a> {
         self
     }
 
+    pub fn with_tpu_version(mut self, use_trading_proto_v2: bool) -> Self {
```

Comment:
the function name feels like we gonna pass a version but then we pass a bool flag